docs/reference/installation.rst

```diff
@@ -212,19 +298,42 @@ Now that your module is generated, you can register it
                             ApplicationSonataMediaBundle: ~
                             SonataMediaBundle: ~
 
+.. note::
+    If you are not using Symfony Flex, this configuration should be added
+    to ``app/config/config.yml``.
+
+You will have to exclude your ``Application`` folder from Symfony service
+autowiring:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # config/services.yaml
+        services:
+            App\:
+                resource: '../src/*'
+                exclude: '../src/{Entity,Tests,Application}'
+
+.. note::
+    If you are not using Symfony Flex, skip this part.
+
 Now, you can build up your database:
 
 .. code-block:: bash
 
-    $ app/console doctrine:schema:[create|update]
+    $ bin/console doctrine:schema:[create|update]
 
 
 If they are not already created, you need to add specific folder to allow uploads from users:
 
 .. code-block:: bash
 
-    $ mkdir web/uploads
-    $ mkdir web/uploads/media
-    $ chmod -R 0777 web/uploads
+    $ mkdir public/uploads
+    $ mkdir public/uploads/media
+    $ chmod -R 0777 public/uploads
```

Comment:
I don't think we should recommend this. Maybe just add a comment instead, because the right way will depend on the installation. Something like "make sure your http user can write to this directory, using `chmod` or `setfacl`